### PR TITLE
Feature working cookiebot

### PR DESCRIPTION
**Pull request by stumpdk**
Added ignore tags to every script on frontpage.

Note that the last commit is untested. This is only included to show how a Cookiebot report page could be created.

### CODE REVIEW

File: app/views/cookieinfo.pug [hunk 1]
```diff
@@ -0,0 +1,3 @@
+extends index
+append javascript
+    script(id="CookieDeclaration" src="https://consent.cookiebot.com/e079a717-1a3f-4fd9-8677-b42d07ce6446/cd.js" type="text/javascript" async)
```
Comment (hypesystem) [inline app/views/cookieinfo.pug:3]: We solved this one slightly differently here: https://github.com/CopenhagenCityArchives/kbh-billeder/pull/143

So I think we should just pull this part out again?

Comment (hypesystem) [reply app/views/cookieinfo.pug:3]: That is, remove the cookieinfo.pug, cookieinfo controller, and route /cookies2.

**Comment by hypesystem** [comment]
I'll just make the changes now -- i have the time :smile:

File: app/views/cookieinfo.pug (hunk 1 above)
Comment (stumpdk) [reply app/views/cookieinfo.pug]: Hi Mads. It would be great if you can do it! _)

**Pull request merged**